internal/github_client/github_client.go

```diff
@@ -4,10 +4,12 @@ import (
 	"context"
 	"fmt"
 	"net/http"
+	"os"
 	"strconv"
 	"time"
 
 	"github.com/bradleyfalzon/ghinstallation/v2"
+	"github.com/golang-jwt/jwt"
```

Comment:
Please use v4 or even v5. 3.2.2 is 3 years old already 🙂

Reply:
Done!